src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/SHA256.cs

```diff
@@ -36,21 +30,22 @@ public Implementation()
                 HashSizeValue = _hashProvider.HashSizeInBytes * 8;
             }
 
-            protected sealed override void HashCore(byte[] array, int ibStart, int cbSize)
-            {
+            protected sealed override void HashCore(byte[] array, int ibStart, int cbSize) =>
                 _hashProvider.AppendHashData(array, ibStart, cbSize);
-            }
 
-            protected sealed override byte[] HashFinal()
-            {
-                return _hashProvider.FinalizeHashAndReset();
-            }
+            protected override void HashCore(ReadOnlySpan<byte> source) =>
```

Comment:
>we should be consistent by removing sealed from all of the methods.

I disagree. It's extra work for the reader to scroll up to see whether the class is sealed. Furthermore, if the class is ever unsealed, every method now becomes unsealed when it should be the opposite: methods should only be unsealed when there's a specific need.